sources/packages/web/src/services/http/ApplicationApi.ts

```diff
@@ -109,9 +109,10 @@ export class ApplicationApi extends HttpBaseClient {
     studentId?: number,
   ): Promise<PaginatedResultsAPIOutDTO<ApplicationSummaryAPIOutDTO>> {
     let url =
-      AuthService.shared.authClientType === ClientIdType.AEST
+      AuthService.shared.authClientType === ClientIdType.AEST ||
+      AuthService.shared.authClientType === ClientIdType.Institution
```

Comment:
Should we base this condition on studentId like we do for most of other places?

Reply:
mm...since it is the API service and technically it's based on the client type (same as we add client root), I don't see an issue here. Because. if it is a nonstudent client and the student is not present, according to this logic it will throw the expected error (student not found, etc..), but if we add the `student id` check, it will throw an authorization error right (because of the wrong token).?

Reply:
When I analyzed, my understanding is different, correct me if wrong.

For this scenario for example, if it is an AEST Client and by mistake studentId is not passed to this method,

it will form the URL like `student/undefined/application-summary`.(I believe parseint pipe will throw bad request error)

If the logic is otherwise(studentId check), and if the same happens URL `student/application-summary` will be formed and returned 404 because any non student endpoint cannot exist without studentId followed by student `student/studentId`.

I was not trying to say to go this way(by studentId way) for a technical reason as I don't see a much tangible difference, my point was as we are doing like this mostly in our application, we can follow the same here unless there is a very strong reason. 

@andrewsignori-aot @andrepestana-aot @guru-aot thoughts?

Reply:
I was on a chat we @ann-aot and we quickly talk about it. I believe that the suggestion is valid @dheepak-aot and we can change it as per your recommendation.

Reply:
make sense after reading the expalination